config/environment.rb

```diff
@@ -3,3 +3,14 @@
 
 # Initialize the Rails application.
 Rails.application.initialize!
+
+# Set ActionMailer to use Sendgrid - CHANGE TO PRODUCTION SETTINGS UPON DEPLOYMENT
+ActionMailer::Base.smtp_settings = {
+  user_name: 'patmbolger',
+  password:  '38LjEPStn-',
+  domain:    'herokuapp.com',
+  address:   'smtp.sendgrid.net',
+  port:       587,
+  authentication: :plain,
+  enable_starttls_auto: true  
+}
```

Comment:
it's preferable to make use of environment variables like this

```ruby
ActionMailer::Base.smtp_settings = {
  user_name: ENV.fetch('SMTP_USERNAME'),
  password:  ENV.fetch('SMTP_PASSWORD'),
  domain:    ENV.fetch('SMTP_DOMAIN'),
  address:   ENV.fetch('SMTP_ADDRESS'),
  port:      ENV.fetch('SMTP_PORT'),
  authentication: :plain,
  enable_starttls_auto: true  
}
```
We can set those on heroku easily